iojs.org/aliases.json

```diff
@@ -48,7 +46,6 @@
     "josh.gavant@gmail.com"
   ] },
   { "from": "security", "to": [
-    "info@bnoordhuis.nl",
```

Comment:
I'd leave this in

AFAIK Ben didn't retire from the security team

/cc @bnoordhuis

Reply:
TSC needs to do something about the arbitrary and largely (completely?) undocumented way we handle membership on that mailing list and the security team.  Added https://github.com/nodejs/TSC/issues/358 to the TSC agenda for next time.